Formula/metis.rb

```diff
@@ -27,8 +22,18 @@ class Metis < Formula
 
   depends_on "cmake" => :build
 
+  resource "gklib" do
+    url "https://github.com/KarypisLab/GKLib.git", revision: "8bd6bad750b2b0d90800c632cf18e8ee93ad72d7"
+  end
+
   def install
-    system "make", "config", "prefix=#{prefix}", "shared=1"
+    args = %W[prefix=#{prefix} shared=1 cc=#{ENV.cc}]
+    resource("gklib").stage do
+      system "make", "config", *args
+      system "make", "install"
+    end
```

Comment:
We should just build this as a static library so we only need to install it somewhere in `buildpath` I think. Unless `metis` headers also need `gklib` headers?